packages/livekit-rtc/src/index.ts

```diff
@@ -17,8 +17,8 @@ export {
 } from './track.js';
 export { VideoFrame } from './video_frame.js';
 export { AudioFrame } from './audio_frame.js';
-export { AudioStream, AudioFrameEvent } from './audio_stream.js';
-export { VideoStream, VideoFrameEvent } from './video_stream.js';
+export { AudioStream, AudioStreamEvent, AudioFrameEvent } from './audio_stream.js';
+export { VideoStream, VideoStreamEvent,VideoFrameEvent } from './video_stream.js';
```

Comment:
I threw these in, too, so you can use the proper types when listening to these events (same as `RoomEvent` above)